test.lisp

```diff
@@ -1,22 +1,87 @@
 (in-package :cl)
 
 (defpackage :inferior-shell-test
-  (:use :cl :inferior-shell :hu.dwim.stefil))
+  (:use :cl :inferior-shell :fiveam))
 
 (in-package :inferior-shell-test)
 
 (declaim (optimize (speed 1) (debug 3) (space 3)))
 
-(defsuite* (test-suite
-            :in root-suite
-            :documentation "Testing inferior-shell"))
+(defmacro with-temp-file (symbol &rest body)
+  `(let ((,symbol (format nil "/tmp/inf-shell-file-~a" (random 100000))))
+     (run/ss `(rm ,,symbol) :on-error nil)
+     (unwind-protect
+          (progn ,@body)
+       (run/ss `(rm ,,symbol) :on-error nil))))
 
-(defun do-test-inferior-shell ()
-  (is (equal (run/ss '(echo (1) "2" (+ 3))) "1 2 3"))
```

Comment:
Why change the order of arguments?

Reply:
I find it easier to read when the target (shorter, e.g. `"1 2 3"`) and the source (longer, e.g. `'(echo (1) "2" (+ 3)))`) are split into different lines. It's also easier to put the shorter form as the zeroth argument of `#'equal` instead of the first.